instalooter/looters.py

```diff
@@ -126,7 +126,7 @@ def _login(cls, username, password, session=None):
         session = cls._init_session(session)
         headers = copy.deepcopy(session.headers)
         homepage = "https://www.instagram.com/"
-        login_url = "https://www.instagram.com/accounts/login/ajax/"
```

Comment:
removing the ajax prefix messes up the login token resulting in error of expecting value: line1 column 1 (char 0)

Reply:
Sorry I am new to this. So what should the login_url be? Will it fix the "......object is not subscriptable" error ?